### PR TITLE
Refactor code according to the Anvar s comments

### PR DESCRIPTION
**Pull request by visiteur85**

### CODE REVIEW

File: src/pages/testPage/index.tsx [hunk 1]
```diff
@@ -37,6 +37,9 @@ export default function TestPage({
     }
     setQuestion(numberOfQuestion + 1);
   }, [numberOfQuestion, questions.length]);
+  const toggleIsRunning = () => {
```
Comment (AnvarAka1) [inline src/pages/testPage/index.tsx:41]: Minor thing, but always use blank line between functions. Make the file clearer for the reader

File: src/store/reducers/auth-reducer.ts [hunk 2]
```diff
@@ -2,13 +2,14 @@ import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
...
 import { authApi } from '../../api/authApi';
-import { LoginFormType, RegistrationFormType } from '../../Types/AuthTypes';
+import { LoginFormType, RegistrationFormType } from '../../types/AuthTypes';
 
 export const registration = createAsyncThunk(
   'registration/register',
   async (data: RegistrationFormType, { rejectWithValue }) => {
     try {
       const response = await authApi.registration(data);
+      localStorage.setItem('token', response.data.accessToken);
```
Comment (AnvarAka1) [inline src/store/reducers/auth-reducer.ts:12]: Do not forget about `setTokenToStorage` function instead of `localStorage.setItem`
Needs abstraction

Comment (visiteur85) [reply src/store/reducers/auth-reducer.ts:12]: Yes, i remember about it, i've created this task.

File: src/pages/testPage/index.tsx (hunk 1 above)
Comment (visiteur85) [reply src/pages/testPage/index.tsx:41]: done

**Review by AnvarAka1 — APPROVED** [review]

**Review by otabek — APPROVED** [review]

**Pull request merged**